roles/common/tasks/main.yml

```diff
@@ -47,6 +47,8 @@
     - openjdk-8-jdk
     - openjdk-8-source
     - zenity
+    - vim
```

Comment:
Minor request: Can `unzip` be above `vim` and `vim` be above `zenity`? This should help in preserving the alphabetical ordering.

Reply:
Can we also add the vim-gnome package below `vim` and before `zenity`?